bot/exts/evergreen/wikipedia.py

```diff
@@ -0,0 +1,84 @@
+import asyncio
+import datetime
+import logging
+from typing import Optional
+
+from discord import Color, Embed, Member
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
+WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+
+
+class WikipediaCog(commands.Cog):
+    """Get info from wikipedia."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.http_session = bot.http_session
+
+    async def search_wikipedia(self, search_term: str) -> Optional[str]:
```

Comment:
```suggestion
    async def search_wikipedia(self, search_term: str) -> Optional[List[str]]:
```
You will want to import `List` from `typing` as well.